data_acquisition/data_acquisition.py

```diff
@@ -23,6 +38,9 @@ def request_ziprecruiter_jobs(page_num, api_key):
     cur = con.cursor()
     # parse ZipRecruiter's JSON response
     for job in response["jobs"]:
+      if not is_internship(job):
```

Comment:
is there a reason you did `if not is_internship(job): continue` instead of `if is_internship(job):`?

Reply:
I personally prefer to do continues/breaks to avoid an extra indentation layer for all the rest of the loop body.